### ISSUE TITLE
Analytics opt out (Firefox) ReferenceError: event is not defined

### ISSUE THREAD

**Issue by mrengy** [department-of-veterans-affairs/vets-website#7569]
Clicked "opt out" on https://www.vets.gov/analytics-opt-out.html, but the button did not change state (expected it to change to say "opted out", which worked for me in Chrome). Getting an error in the console "ReferenceError: event is not defined <anonymous> https://www.vets.gov/analytics-opt-out.html:327:183" as shown in the attached screenshot. 
<img width="1278" alt="screen shot 2018-04-12 at 11 35 24 am" src="https://user-images.githubusercontent.com/2223603/38688041-1d5a2e78-3e46-11e8-9230-3fd4696727e8.png">

Mac OS 10.12.6, Firefox 59.0.2 (64-bit)

**Comment by dmethvin-gov** [comment]
The original source for these pages is written in Markdown, then translated to HTML and the [event handlers are transformed](https://github.com/department-of-veterans-affairs/vets-website/blob/32e40e4ed64f957f949bc248d188899567d0eb19/src/site/stages/build/plugins/nonceTransformer.js#L38) at that time. The generated code there passes `ev` for the name of the event object in the handler. In [analytics-opt-out.md](https://github.com/department-of-veterans-affairs/vagov-content/blob/15a90d8cb65883200e88dcde0353d4d780175b41/pages/analytics-opt-out.md), the handler code there uses `event.target` and not `ev.target`. (You'll have to [look at the raw Markdown](https://raw.githubusercontent.com/department-of-veterans-affairs/vagov-content/15a90d8cb65883200e88dcde0353d4d780175b41/pages/analytics-opt-out.md) to see it). That is what causes the error in Firefox.

The reason this doesn't show an error in Chrome is because Chrome implements the [`window.event` object compat](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) that was introduced by IE many moons ago. So it exposes `event` and the code works fine there. Firefox only does it behind a flag because some sites use the presence of `window.event` to determine whether they're on IE and try to run different code. 

Bottom line, this will be fixed by changing `ev` to `event` in [analytics-opt-out.md](https://github.com/department-of-veterans-affairs/vagov-content/blob/15a90d8cb65883200e88dcde0353d4d780175b41/pages/analytics-opt-out.md). AFAICT [this is the only page on vagov-content](https://github.com/department-of-veterans-affairs/vagov-content/search?q=onclick) that uses the `ev` variable and it makes more sense to rename it to `event`.
